src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -2,7 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import IconColumnSelector from '@carbon/icons-react/lib/column/20';
 import IconFilter from '@carbon/icons-react/lib/filter/20';
-import { DataTable, Button } from 'carbon-components-react';
+import IconEdit from '@carbon/icons-react/lib/edit/20';
```

Comment:
There is now an updated entrypoint for the icons library to eliminate the long import paths.

```diff
- import IconEdit from '@carbon/icons-react/lib/edit/20';
+ import IconEdit20 from '@carbon/icons-react';
```

Reply:
Something bad happened when I tried the updated entrypoint way, so I undid it and waiting for your comments.
![image](https://user-images.githubusercontent.com/37580544/70776482-2038c200-1db8-11ea-8784-df260a91e38d.png)

Reply:
You need to wrap it in curly braces `import { IconEdit20 } from '@carbon/icons-react';` I believe.